scripts/build-docs-content.ts

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env node
+
+/**
+ * Script that builds the docs content NPM package and moves it into a conveniently
+ * accessible distribution directory (the project `dist/` directory).
+ */
+
+import {cd, chmod, cp, exec, mkdir, rm, set} from 'shelljs';
```

Comment:
👀

Reply:
seems sorted 😄 not sure what you expect here. The VSCode plugin seems to leverage some ESLint sort "algorithm".